package.json

```diff
@@ -43,7 +43,7 @@
     "core-js": "^2.5.4",
     "d3": "4.13.0",
     "fix": "0.0.6",
-    "ids-enterprise": "4.19.0-dev.20190513",
+    "ids-enterprise": "^4.19.0-dev.20190513",
```

Comment:
Can we make this 14 now? I wanted last nights build on anyways.

Reply:
@tmcconechy I've just released a NG dev version `5.4.0-dev.20190514` that depends on `4.19.0-dev.20190514`.

Reply:
I can merge it when you've done your PR, if you like.